security/tor/src/opnsense/mvc/app/models/OPNsense/Tor/General.xml

```diff
@@ -99,5 +99,23 @@
       <default>0</default>
       <Required>Y</Required>
     </dns_map_hosts>
+    <client_authentications>
+      <client_auth type="ArrayField">
+        <enabled type="BooleanField">
+          <default>1</default>
+          <Required>Y</Required>
+        </enabled>
+        <onion_service type="TextField">
+          <Required>Y</Required>
+          <default>exampleexample23.onion</default>
```

Comment:
the default is weird here

Reply:
better idea? it has to have an exact length and needs to consist of characters a subset of numbers.